config/initializers/bulkrax.rb

```diff
@@ -25,96 +25,91 @@
-  config.field_mappings["Bulkrax::CsvParser"]["contributing_institution"] = { from: ["contributing_institution"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["extent"] = { from: ["extent"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["format_digital"] = { from: ["format", "format_digital"], parsed: true, split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["format_original"] = { from: ["format_original"], split: /\s*[;]\s*/, parsed: true }
-  config.field_mappings["Bulkrax::CsvParser"]["has_manifest"] = { from: ["has_manifest"] }
-  config.field_mappings["Bulkrax::CsvParser"]["place"] = { from: ["place"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["remote_files"] = { from: ["thumbnail_url", "remote_files"], parsed: true }
-  config.field_mappings["Bulkrax::CsvParser"]["remote_manifest_url"] = { from: ["remote_manifest_url"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["rights_holder"] = { from: ["rights_holder"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["rights_statement"] = { from: ["rights", "rights_statement"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["time_period"] = { from: ["time_period"], split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::CsvParser"]["types"] = { from: ["type", "types", "resource_type"], split: /\s*[;]\s*/, parsed: true }
+  config.field_mappings['Bulkrax::OaiOmekaParser'] = default_field_mapping.merge({
+    # add, remove or override custom mappings for this parser here
+  })
 
-  # internet archive - custom mappings - exclude contributor, add date parser
-  config.field_mappings["Bulkrax::OaiDcParser"].each {|key,value| config.field_mappings["Bulkrax::OaiIaParser"][key] = value }
-  config.field_mappings["Bulkrax::OaiIaParser"]["contributor"] = { from: ["contributor"], excluded: true}
-  config.field_mappings["Bulkrax::OaiIaParser"]["date"] = { from: ["date"], parsed: true, split: /\s*[;]\s*/}
-  config.field_mappings["Bulkrax::OaiIaParser"]["format_digital"] = { from: ["format"], excluded: true }
+  config.field_mappings['Bulkrax::OaiPtcParser'] = default_field_mapping.merge({
+    'format_original' => { from: ['format'], parsed: true, split: /\s*[;]\s*/ }
+  })
+  # switch format_digital for format_original
+  config.field_mappings['Bulkrax::OaiPtcParser'].delete('format_digital')
 
-  # ptc - custom mappings - switch format_digital for format_original
-  config.field_mappings["Bulkrax::OaiDcParser"].each {|key,value| config.field_mappings["Bulkrax::OaiPtcParser"][key] = value }
-  config.field_mappings["Bulkrax::OaiPtcParser"]["format_original"] = { from: ["format"], parsed: true, split: /\s*[;]\s*/ }
-  config.field_mappings["Bulkrax::OaiPtcParser"].delete("format_digital")
+  config.field_mappings['Bulkrax::OaiSetsParser'] = default_field_mapping.merge({
+    # add, remove or override custom mappings for this parser here
+  })
```

Comment:
```suggestion
  # config.field_mappings['Bulkrax::OaiSetsParser'] = default_field_mapping.merge({
    # add, remove or override custom mappings for this parser here
  # })
```

Reply:
refer to the [above comment](https://github.com/scientist-softserv/atla_digital_library/pull/294#discussion_r1184341602).